docs/archetypes/aks/overview.adoc

```diff
@@ -0,0 +1,481 @@
+
+| `${general.workload.kubernetes.service-principal-client-id}`
+| Unique to AAD tenant
+| ID of the Service Principal used to represent the identity of the AKS cluster. This SPN is used by the cluster to provision resources through ARM. Steps on creating this SPN can be found https://docs.microsoft.com/azure/aks/kubernetes-service-principal#manually-create-a-service-principal[here].
+
+| `${general.workload.kubernetes.service-principal-client-secret}`
+|
+| Secret for the previously created AAD service principal.
+
+| `${general.workload.kubernetes.rbac-server-appid}`
+| Unique to AAD tenant
+| ID for server application created for integration with Azure AD. A reference for creating this identity can be found https://docs.microsoft.com/azure/aks/aad-integration#create-server-application[here]. 
+
+| `${general.workload.kubernetes.rbac-server-secret}`
+|
+| Secret for the previously created AAD server application.
+
+| `${general.workload.kubernetes.rbac-client-appid}`
+| Unique to AAD tenant
+| ID for client application created for integration with Azure AD. A reference for creating this identity can be found https://docs.microsoft.com/azure/aks/aad-integration#create-client-application[here]. 
+
+| `${general.workload.kubernetes.rbac-tenant}`
+| Globally unique
+| AAD Tenant where users and groups are stored that are allowed to interact with the AKS cluster through kubernetes RBAC integration. A reference can be found https://docs.microsoft.com/azure/aks/aad-integration[here]. 
+
+| `${general.workload.kubernetes.cluster-admin-group-id}`
+| Unique to RBAC AAD tenant
+| Cluster Admin persona AAD group in the RBAC tenant (used to manage group membership). A description of each persona can be found in the link:overview.adoc#deployment-assumptions[deployment assumptions] section of this document. 
+
+| `${general.workload.kubernetes.noc-user-group-id}`
+| Unique to RBAC AAD Tenant
+| NOC persona AAD group in the RBAC Tenant (used to manage group membership). A description of each persona can be found in the link:overview.adoc#deployment-assumptions[deployment assumptions] section of this document. 
+
+| `${general.workload.kubernetes.dev-user-group-id}`
+| Unique to RBAC AAD Tenant
+| Developer persona AAD group in the RBAC Tenant (used to manage group membership). A description of each persona can be found in the link:overview.adoc#deployment-assumptions[deployment assumptions] section of this document. 
+|===
+
+== Run the deployment
+
+=== Single step archetype deployment - Required permissions: Subscription Owner
+
+This section performs an end-to-end deployment of the AKS archetype. It assumes the account performing the deployment has *Owner* permissions to the subscription where the archetype is being deployed. 
+
+Start the workload deployment by running the following commands based on the platform you are deploying from:
+
+.Docker
+[source,bash]
+python vdc.py create workload -path archetypes/aks/archetypes.json
```

Comment:
should this be archetypes.json, or something else?

Reply:
The file name is changed.